### ISSUE TITLE
Reproduce Training GCNv2

### ISSUE THREAD

**Issue by Crazylov3** [jiexiong2016/GCNv2_SLAM#72]
Do anyone try to re-train GCNv2? I try to train this model from scratch but I can't reproduce the result. I have a problem using straight-through estimator (which they mentioned in paper) to training with binary description, the loss does not converge Any suggests, pls

**Comment by yhjsss** [comment]
Hello, I am learning relevant knowledge, but I encountered many problems when reproducing it. Can you share your training code? Thank you very much.

**Comment by mono0826** [comment]
> Hello, I am learning relevant knowledge, but I encountered many problems when reproducing it. Can you share your training code? Thank you very much.

Hello,Did you reproduce the training code?
